resources/lib/WonderfulSubsBrowser.py

```diff
@@ -301,39 +308,39 @@ def get_by_letter(self, letter, page = 1):
             "count": self._RESULTS_PER_SEARCH_PAGE,
             "index": (page-1) * self._RESULTS_PER_SEARCH_PAGE,
         }
-        url = self._to_url("api/media/all")
+        url = self._to_url("{}/media/all".format(API_BASE))
         return self._process_anime_view(url, data, "letter/%s/%%d" % letter, page)
 
     def get_all(self,  page=1):
         data = {
             "count": self._RESULTS_PER_SEARCH_PAGE,
             "index": (page-1) * self._RESULTS_PER_SEARCH_PAGE,
         }
-        url = self._to_url("api/media/all")
+        url = self._to_url("{}/media/all".format(API_BASE))
         return self._process_anime_view(url, data, "all/%d", page)
 
     def get_popular(self,  page=1):
         data = {
             "count": self._RESULTS_PER_SEARCH_PAGE,
             "index": (page-1) * self._RESULTS_PER_SEARCH_PAGE,
         }
-        url = self._to_url("api/media/popular")
+        url = self._to_url("{}/media/popular".format(API_BASE))
         return self._process_anime_view(url, data, "popular/%d", page)
 
     def get_latest(self, page=1):
         data = {
             "count": self._RESULTS_PER_SEARCH_PAGE,
             "index": (page-1) * self._RESULTS_PER_SEARCH_PAGE,
         }
-        url = self._to_url("api/media/latest")
+        url = self._to_url("{}/media/latest".format(API_BASE))
         return self._process_anime_view(url, data, "latest/%d", page)
 
     def get_random(self, page=1):
         data = {
             "count": self._RESULTS_PER_SEARCH_PAGE,
             "index": (page-1) * self._RESULTS_PER_SEARCH_PAGE,
         }
-        url = self._to_url("api/media/random")
+        url = self._to_url("{}/media/random".format(API_BASE))
```

Comment:
I didn't want to change the base class behaviour, because I didn't know, where it was used. But I'll just set ```_BASE_URL``` to BASE_URL + API_BASE and we should be set :)